tests/test_new_meshing_workflow.py

```diff
@@ -634,7 +635,7 @@ def test_watertight_workflow_dynamic_interface(mixing_elbow_geometry, new_mesh_s
         watertight.create_volume_mesh
     assert (
         msg.value.args[0]
-        == "'EnhancedMeshingWorkflow' object has no attribute 'create_volume_mesh'"
+        == "'WatertightMeshingWorkflow' object has no attribute 'create_volume_mesh'"
```

Comment:
- We don't want such assertions.
- `pytest.raises()` has an optional `match` argument which supports regex.
- From Python 3.10, AttributeError has `obj` and `name` attributes which can be checked directly.

Reply:
- regex approah might be best right now as it fits neatly into the pytest framework, thereby eliminating boilerplate.

Reply:
Done something with this already, but will look into it further.

Reply:
Will remove the error msg string check.

Reply:
Done.